src/codewhisperer/commands/basicCommands.ts

```diff
@@ -35,21 +39,31 @@ export const toggleCodeSuggestions = Commands.declare(
         })
     }
 )
+/* 
+createGettingStartedNode(Learn) will be a childnode of CodeWhisperer
+onClick on this "Learn" Node will open the Learn CodeWhisperer Page.
+*/
+export const createGettingStartedNode = () =>
+    CodeWhispererCommandDeclarations.instance.declared.showGettingStartedPage
+        .build('codewhispererDeveloperTools')
+        .asTreeNode({
+            label: localize('AWS.explorerNode.codewhispererGettingStartedNode.label', 'Learn'),
+            iconPath: getIcon('aws-codewhisperer-learn'),
+        })
 
 export const enableCodeSuggestions = Commands.declare(
     'aws.codeWhisperer.enableCodeSuggestions',
     (context: ExtContext) => async () => {
         await set(CodeWhispererConstants.autoTriggerEnabledKey, true, context.extensionContext.globalState)
         await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
         await vscode.commands.executeCommand('aws.codeWhisperer.refresh')
+        const prompts = PromptSettings.instance
 
-        const hasShownWelcomeMsgBefore = get(
-            CodeWhispererConstants.welcomeMessageKey,
-            context.extensionContext.globalState
-        )
-        if (!hasShownWelcomeMsgBefore) {
-            showCodeWhispererWelcomeMessage(context)
-            await set(CodeWhispererConstants.welcomeMessageKey, true, context.extensionContext.globalState)
+        const shouldShow = await prompts.isPromptEnabled('codeWhispererNewWelcomeMessageKey')
```

Comment:
"key" is redundant in the name.
```suggestion
        const shouldShow = await prompts.isPromptEnabled('codeWhispererNewWelcomeMessage')
```

Reply:
Updated Accordingly